mobile/src/board/components/BoardView/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Board, Player, GameState } from "../../entity";
```

Comment:
I'm not a fan of having a gazillion files named "index.tsx" - it really becomes a pain to find/navigate between files in the IDE